judgments/templatetags/query_filters.py

```diff
@@ -16,11 +16,46 @@ def make_query_string(params):
     return "&".join(pairs)
 
 
+@register.filter
+def removable_filter_param(key):
+    excluded = [
+        "order",
+        "per_page",
+        "court",
+        "from_day",
+        "from_month",
+        "from_year",
+        "to_day",
+        "to_month",
+        "to_year",
+        "text_date_input",
+    ]
+    return key not in excluded
+
+
+def date_filter_param(key):
```

Comment:
Could you add something describing what the intention is here? I'm not sure that these are `param`s in the same sense as `to_month` above; one is input, the other a gestalt entity.

I think I get it -- if a 'date filter parameter' is removed, we remove all the subcomponents -- but this is quite subtle and might appreciate a little documentation.